src/plugins/discord/fetcher.js

```diff
@@ -83,7 +83,6 @@ export class Fetcher {
         bot: x.user.bot || x.user.system || false,
       },
       nick: x.nick || null,
-      roles: x.roles,
```

Comment:
Since #1733 was merged, can you rebase so the role removing isn't included here?

Reply:
Updated